package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "lightningdevkit.org",
-  "version": "1.0.0",
+  "name": "payjoindevkit.org",
+  "version": "0.0.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "name": "lightningdevkit.org",
-      "version": "1.0.0",
+      "name": "payjoindevkit.org",
+      "version": "0.0.1",
       "license": "MIT",
```

Comment:
is this a manual change? I don't see a change causing the lock file to change from the package.json

Reply:
It got auto-updated as a result of running `npm install`. The same thing happens on the main branch, so it likely slipped through the cracks in a previous commit.